app/components/auk/navbar.js

```diff
@@ -1,25 +1,21 @@
 import Component from '@glimmer/component';
 
+/**
+ *
+ * @argument {String} border Can be "top", "bottom" (default) or "none"
+ * @argument {String} skin Can be "gray-100", "gray-200" or "white (default)"
+ * @argument {String} size
+ * @argument {Boolean} auto TODO: document what this does
```

Comment:
TODO, die auto is voor de hoogte, anders heb je min-height van 5rem
```
.auk-navbar--auto {
  height: auto;
  min-height: auto;
}
```

Reply:
styleguide toolbar sample 5 heeft dat, maar de css uitzetten geeft geen verschil :D 
https://kaleidos-dev.vlaanderen.be/styleguide/toolbar